tests/test_telescope_loader.py

```diff
@@ -662,6 +662,37 @@ def test_config_fea_file_num():
 
         assert configd == reference
 
+def test_config_focus_z():
+    """Test that focusZ is properly set from configs."""
+
+    telescope = imsim.load_telescope("LSST_r.yaml")
+    shifted_ref = (telescope
+        .withLocallyShiftedOptic('LSSTCamera', [0.0, 0.0, 1e-3])
+    )
+
+    focusz_shifted_function_call = imsim.load_telescope("LSST_r.yaml", focusZ=1e-3)
+
+    assert shifted_ref == focusz_shifted_function_call
+
+    shifted_config = textwrap.dedent(
+            """
+            input:
+                telescope:
+                    file_name: LSST_r.yaml
+                    focusZ : 1e-3
+            """
+        )
+
+    config = yaml.safe_load(shifted_config)
+    galsim.config.ProcessInput(config)
+    focusz_shifted_config = galsim.config.GetInputObj(
+        'telescope',
+        config['input']['telescope'],
+        config,
+        'telescope'
+    ).fiducial
+
+    assert shifted_ref == focusz_shifted_config
```

Comment:
This test doesn't seem to use the image you added to tests/data.  Do we need to add that to the repo?

Reply:
I added `FOCUSZ` to the headers in the existing file because without it multiple tests in `test_readout.py` fail.

Reply:
Oh, I see.  I thought it was a new file.  It's just changes to the header in the existing one.  Got it.